src/autoschedulers/anderson2021/scripts/utils.sh

```diff
@@ -0,0 +1,323 @@
+#!/bin/bash
+
+function make_dir_path_absolute() {
+    local -r path=$1
+    local -n absolute_path_ref=$2
```

Comment:
This will fail on recent macOS as they've switched to zsh as their default, which doesn't support '-n'. Consider adding another warning for the user?